### DISCUSSION TITLE
Podwójne wpisy lekcji.

### DISCUSSION THREAD

**Discussion by tiwek** [Czapla-dev/vulcan-for-hassio#25] [Ideas]
Taka sytuacja.
Szkoła integracyjna, na części lekcji dodatkowo jest nauczyciel wspomagający.
W kalendarzu wygląda to tak:

![image](https://user-images.githubusercontent.com/74269691/156352956-064ec324-a3aa-494c-9f3c-9a61679b6ad7.png)

Czy można by się jakoś pozbyć tego drugiego wpisu ??
Jakiś plik z nazwami przedmiotów które trzeba pomijać ??

**Comment by Antoni-Czaplicki** [comment]
Nie mam teraz czasu aby tym się zająć, a poza tym ten problem dotyczy niewielu więc nie chcę dodawać dodatkowego pliku z pomijanymi lekcjami. Jeśli masz ochotę możesz zrobić to u siebie szybko i lokalnie wpisując dane bezpośrednio w kod integracji bądź trochę bardziej zaawansowane - używając opcji integracji i otworzyć PR z tymi zmianami.
